packages/grid/stories/grid-sortColumns.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { CellEditor, DropdownCellEditor, Grid, GridColumn } from "../src";
+import "./grid.stories.css";
+import { Story } from "@storybook/react";
+import { createDummyInvestors, Investor, investorKeyGetter } from "./dummyData";
+
+export default {
+  title: "Data Grid/Data Grid",
+  component: Grid,
+  argTypes: {},
+};
+
+const dummyInvestors = createDummyInvestors();
+
+const SortColumnsTemplate: Story<{}> = () => {
+  const scoreOptions = ["-", "5%", "10%", "15%", "20%"];
+  const onScoreChange = (row: Investor, rowIndex: number, value: string) => {
+    dummyInvestors[rowIndex].score = value;
+  };
+
+  return (
+    <Grid
+      rowData={dummyInvestors}
+      rowKeyGetter={investorKeyGetter}
+      style={{ height: 600 }}
+      headerIsFocusable
+      className="grid"
+      zebra={true}
+      columnSeparators={true}
+    >
+      <GridColumn
+        name="Name"
+        id="name"
+        defaultWidth={200}
+        getValue={(rowData: Investor) => rowData.name}
+        isSortable
+      />
+      <GridColumn
+        name="Strategy"
+        id="strategy"
+        getValue={(rowData: Investor) => rowData.strategy.join(", ")}
+        isSortable
+      />
+      <GridColumn
+        name="Date incorporated"
+        id="date"
+        defaultWidth={150}
+        getValue={(rowData: Investor) => rowData.date}
+        isSortable
+      />
+      <GridColumn
+        name="Score"
+        id="score"
+        getValue={(r) => r.score}
+        onChange={onScoreChange}
+        isSortable
+        customSort={({ rowData, sortOrder }) => {
```

Comment:
It would be good to create an example that focuses on data that isn't local and how that interaction would work

Reply:
@joshwooding good idea - have added that in latest commit. Do let me know if it's similar to what you had in mind

Reply:
This example still shows sorting on the client. Even though the initial data is fetched with useSWR, the sort doesn't re-fetch the data from the server sorted but instead works on data it already has locally.